docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless Preview 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# FeatureBase Serverless
+
+{: .no_toc }
+
+{% include /serverless/serverless-preview-warning.md %}
+
+FeatureBase Serverless is a new architecture for FeatureBase that separates compute power and storage and allows each to scale independently.
+
+{% include page-toc.md %}
+
+## Why Serverless?
+
+### Separation of storage and compute
+Serverless databases have durable storage that is separated from compute. You can consume and scale each independently based on your use-cases’ needs.
+
+### Zero down time scaling
+Serverless databases can have compute scaled up and down at any time without interruption to your services. Your database can increase the number of [workers](#workers) for surges in traffic during peak hours, as well as scale workers down during quiet hours. Your storage will grow with you over time, without any ceilings or need for manual scaling.
```

Comment:
Don't do single word anchor text for hyperlinks. It won't scan well for the user plus it won't be picked up meaningfully by search

See the help-on-help in the repo for hyperlinks:

/help-on-help/writing-help/writing-hyperlinks.md